react/CozyDialogs/useCozyDialog.js

```diff
@@ -1,7 +1,21 @@
 import { useState } from 'react'
+import { makeStyles } from '@material-ui/core/styles'
+
 import useBreakpoints from '../hooks/useBreakpoints'
 import DialogTransition from './DialogTransition'
 
+const useStyles = makeStyles(theme => {
+  return {
+    dialogTitle: {
+      width: ({ onClose }) =>
+        onClose ? 'calc(100% - 88px)' : 'calc(100% - 64px)',
+      [theme.breakpoints.down('sm')]: {
+        width: 'calc(100% - 32px)'
+      }
+    }
+  }
+})
```

Comment:
The style related to our Dialog is already defined in theme.jsx. We should follow this path. Now, with this PR we have several files to look at. We should avoid that. 

Also, can we know where these values (88 / 64 / 32) come from? Should they be related / linked to other margins / variables?

Reply:
> The style related to our Dialog is already defined in theme.jsx. We should follow this path. Now, with this PR we have several files to look at. We should avoid that.

Oui mais la taille dépend d'une prop, on ne peut pas faire autrement si ?

Reply:
If we take a look at how we handle that for now: 

- We add a classname : https://github.com/cozy/cozy-ui/blob/master/react/CozyDialogs/IllustrationDialog.jsx#L43
- We apply a style depending of the classname https://github.com/cozy/cozy-ui/blob/master/react/MuiCozyTheme/theme.jsx#L505 

no?